openhtf/capabilities/usb/adb_message.py

```diff
@@ -54,8 +54,7 @@
                       'Set to True to save all incoming and outgoing '
                       'AdbMessages and print them on Close().')
 FLAGS = gflags.FLAGS
-_LOG = logging.getLogger('adb_protocol')
```

Comment:
Let's all chat about the various sources of logs we have and establish a taxonomy of sorts for them, I think it's more complicated than simply changing everything to use the same 'HTFTest' logger.  Notably, some logs go into the TestRecord and some don't, so at the very least we need to differentiate those.